interview-questions.md

```diff
@@ -7,44 +7,58 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. In a model called Animal that has_many Sightings, what is the name of the foreign key? Would the foreign key be part of the Animal model or the Sightings model?
 
-  Your answer:
+  Your answer: The foreign key should be named animal_id. It would be part of the Sightings model. 
 
-  Researched answer:
+  Researched answer: Pretty sure this right.
 
 
 
 2. Which RESTful routes must always be passed params? Why?
 
-  Your answer:
+  Your answer: I'm unsure and would need too look that up. My best guesses are as follows: I believe GET would need params to display a specific object but might not need them to display all objects. POST would need them if the object created had required fields. PUT would need them to know which object to replace. PATCH would need them to know which object to update. DELETE would them to know which object to delete.  
 
```

Comment:
Your researched answer is great. I could see why you thought the post would require params since the create method does take strong params, but the route would not.